Lib/fontbakery/profiles/universal.py

```diff
@@ -2212,6 +2213,189 @@ def com_google_fonts_check_math_signs_width(ttFont):
         yield PASS, "Looks good."
 
 
+@check(
+    id="com.google.fonts/check/tabular_kerning",
+    rationale="""
+        Tabular glyphs should not have kerning, as they are meant to be used in tables.
+
+        This check looks for kerning in:
+        - all glyphs in a font in combination with tabular numerals;
+        - tabular symbols in combination with tabular numerals.
+
+        "Tabular symbols" is defined as:
+        - for fonts with a "tnum" feature, all "tnum" substitution target glyphs;
+        - for fonts without a "tnum" feature, all glyphs that have the same width
+        as the tabular numerals, but limited to numbers, math and currency symbols.
+
+        This check may produce false positives for fonts with no "tnum" feature
+        and with equal-width numerals (and other same-width symbols) that are
+        not intended to be used as tabular numerals.
```

Comment:
If the test can have false positives, then it shouldn’t be producing FAIL’s.

Reply:
That's true, but try to examine the problem with me first:

A font with no tnum feature has tabular glyphs. Should they have kerning or can it be assumed that they shouldn't have kerning? I think the latter is the case.